lib/ReactViews/BottomDock/Timeline/DateTimePicker.jsx

```diff
@@ -175,12 +175,10 @@ const DateTimePicker = createReactClass({
 
     renderDayView(datesObject) {
       if (datesObject[this.state.year][this.state.month].dates && datesObject[this.state.year][this.state.month].dates.length > 31) {
-        // Create one date object per day, using an arbitrary time. This does it via Object.keys and moment().
+        // Create one date object per day, using an arbitrary time. This does it via Object.keys.
         const days = datesObject[this.state.year][this.state.month].indice;
-        const daysToDisplay = days.map(d => moment().date(d).month(this.state.month).year(this.state.year));
-        const selected = defined(this.state.day) ? moment().date(this.state.day).month(this.state.month).year(this.state.year) : null;
-        // Aside: You might think this implementation is clearer - use the first date available on each day.
-        // However it fails because react-datepicker actually requires a moment() object for selected, not a Date object.
+        const daysToDisplay = days.map(d => parse(`${d}/${this.state.month}/${this.state.year}`, 'dd/MM/yyyy', new Date()));
```

Comment:
Consider using `new Date(this.state.year, this.state.month - 1, d)` rather than parsing strings.